Manifest.toml

```diff
@@ -0,0 +1,98 @@
+# This file is machine-generated - editing it directly is not advised
```

Comment:
Why is `Manifest.toml` added in the second commit, not the first?

Reply:
I messed up the git history, should be fixed now. Thanks.